paimon-core/src/main/java/org/apache/paimon/manifest/ManifestFileMetaSerializer.java

```diff
@@ -35,7 +35,7 @@ public ManifestFileMetaSerializer() {
 
     @Override
     public int getVersion() {
-        return 2;
+        return 3;
```

Comment:
This version cannot be changed, this PR is a compatible changing, we should keep old version for old readers.

Can you add tests for this?